packages/backend/src/tokens/request.ts

```diff
@@ -109,13 +115,15 @@ export async function authenticateRequest(
 
     let sessionToken = '';
     cookiesToSet.forEach((x: string) => {
-      headers.append('Set-Cookie', x);
-      if (x.startsWith(`${constants.Cookies.Session}=`)) {
-        sessionToken = x.split(';')[0].substring(10);
-      }
+      cookiePipeline.invoke(x).forEach(xVariants => {
+        if (xVariants.startsWith(`${constants.Cookies.Session}=`)) {
+          sessionToken = x.split(';')[0].split('=')[1];
+        }
+        headers.append('Set-Cookie', xVariants);
+      });
```

Comment:
I wish there was a way to return the session token in the handshake payload that was more directly accessible, e.g. a separate key `handshakePayload.sessionToken`, but that would bloat the payload size 😢